tabs/src/components/WalletTransactionLog.tsx

```diff
@@ -62,34 +66,41 @@ const WalletTransactionLog: React.FC<WalletTransactionLogProps> = ({
       console.log('Current user: ', currentUserLNbitDetails);
 
       if (currentUserLNbitDetails && currentUserLNbitDetails.length > 0) {
-        const wallets = await getUserWallets(
-          adminKey,
-          currentUserLNbitDetails[0].id,
-        ); // We'll just look at the private wallets.
-
-        // Loop through all the wallets
-        if (wallets) {
-          console.log('Wallets1');
-          for (const wallet of wallets) {
-            const zaps = await getWalletTransactionsSince(
-              wallet.inkey,
-              paymentsSinceTimestamp,
-              null, //{ tag: 'zap' }
-            );
-
-            for (const zap of zaps) {
-              zap.extra.from = users.filter(
-                u => u.id === zap.extra?.from?.user,
-              )[0];
-              zap.extra.to = users.filter(u => u.id === zap.extra?.to?.user)[0];
-            }
-
-            allZaps = allZaps.concat(zaps);
-            console.log('Zaps: ', allZaps);
+        let inkey: any = null;
+
+        if (activeWalletTabName === 'Your wallet') {
+          inkey = currentUserLNbitDetails[0].privateWallet?.inkey;
+        } else {
+          inkey = currentUserLNbitDetails[0].allowanceWallet?.inkey;
+        }
+
+        if (inkey) {
+          const zaps = await getWalletTransactionsSince(
```

Comment:
Change `zaps` to `transactions` because zaps are just a specific type of transaction with `extra { type: 'zap' }` and you are going to get all types of transactions back here.

Reply:
Done